src/Blosc.jl

```diff
@@ -161,16 +161,26 @@ end
 # Allow Blosc to set the optimal blocksize (default)
 set_default_blocksize() = set_blocksize(0)
 
+if VERSION < v"0.5-"
+    _bytestring = bytestring
+else
+    _bytestring = unsafe_string
+end
```

Comment:
This shouldn't be necessary.   If you update the REQUIRE file to list `Compat 0.8`, then you can just call `unsafe_string` (which is defined in older versions by Compat).